solidity/contracts/token/TBTC.sol

```diff
@@ -3,7 +3,7 @@
 pragma solidity 0.8.4;
 
 import "@thesis/solidity-contracts/contracts/token/ERC20WithPermit.sol";
-import "./MisfundRecovery.sol";
+import "@thesis/solidity-contracts/contracts/token/MisfundRecovery.sol";
 
 contract TBTC is ERC20WithPermit, MisfundRecovery {
```

Comment:
Can we add a simple happy-path test case to check that misfund recovery is possible for `TBTC` contract? I know it's an external contract tested elsewhere but we should check that `TBTC` contract gives a basic functionality expected after extending the `MisfundRecovery` contract.

Reply:
Shouldn't we test ERC20 functions then?

Reply:
Aren't we testing them? I think we should have very simple tests for them too.

Reply:
I think we should rely on PR review for it, especially that post-deployment contracts are immutable. Green build with simple tests would not be enough - one could overwrite some functions from `ERC20WithPermit` and simple tests might not be able to catch the problem.

Reply:
Let's agree to disagree, I won't block this PR.